src/ert/scheduler/lsf_driver.py

```diff
@@ -123,12 +125,36 @@ async def submit(
         /,
         *args: str,
         name: str = "dummy",
-        runpath: Optional[str] = None,
+        runpath: Optional[Path] = None,
     ) -> None:
+        if runpath is None:
+            runpath = Path.cwd()
+
         arg_queue_name = ["-q", self._queue_name] if self._queue_name else []
 
+        script = (
+            "#!/usr/bin/env bash\n"
+            f"cd {shlex.quote(str(runpath))}\n"
+            f"exec -a {shlex.quote(executable)} {executable} {shlex.join(args)}\n"
```

Comment:
So this script is to make sure ">" in `executable` is handled correctly?

Reply:
Yes, it will guarantee that regardless of any bugs in args handling in qsub/bsub.